pkgs/misc/vencord/replace-git.patch

```diff
@@ -1,17 +1,17 @@
 diff --git a/scripts/build/common.mjs b/scripts/build/common.mjs
-index 7ff599a..85b3bfa 100644
+index b63ea61..10e4ee1 100644
 --- a/scripts/build/common.mjs
 +++ b/scripts/build/common.mjs
-@@ -24,7 +24,7 @@ import { promisify } from "util";
- 
+@@ -33,7 +33,7 @@ export const VERSION = PackageJSON.version;
+ export const BUILD_TIMESTAMP = Date.now();
```

Comment:
Correct me if I'm wrong, but if this runs at build time, it doesn't seem [reproducible](https://reproducible-builds.org/)?

Could you use [ `SOURCE_DATE_EPOCH`](https://reproducible-builds.org/docs/source-date-epoch/) from the environment instead? I think it's what you want. (And that sounds like a patch you could send upstream too - `SOURCE_DATE_EPOCH` with `Date.now()` as fallback when not set.)